src/CoreWCF.RabbitMQ/src/CoreWCF/Channels/RabbitMqTransportBindingElement.cs

```diff
@@ -68,20 +68,29 @@ public override string Scheme
         public override long MaxReceivedMessageSize { get; set; }
 
```

Comment:
This should have a default value otherwise it will be 0. This is important for when using a CustomBinding